client/packages/common/src/intl/locales/en/reports.json

```diff
@@ -1,18 +0,0 @@
-{
-    "description.notification-preferences": "By default, the system will tell you if you have more than 3 or 6 months of stock on the homepage dashboard.",
```

Comment:
Since this file already existed, have you thought about it being used already ? 
And apart from relocating the 'common', what else would need to be done ?

Typically when refactoring/renaming we should look at all of the instances where the renamed 'thing' is used. In rust type safety would be guaranteed, but in typescript, unfortunately, it's not entirely so.

Reply:
Good point thanks @andreievg 

I'm just thinking now a possible missed use case is duplicate keys where reports.json was previously used.

I will check now

Reply:
Possibly complicated by mechanisms from this PR:

https://github.com/msupply-foundation/open-msupply/pull/4617